test/spec/Feature/Query/PlanSpec.hs

```diff
@@ -33,6 +33,7 @@ spec actualPgVersion = do
 
       liftIO $ do
         resHeaders `shouldSatisfy` elem ("Content-Type", "application/vnd.pgrst.plan+json; for=\"application/json\"; charset=utf-8")
+        resHeaders `shouldSatisfy` notZeroContentLength
```

Comment:
It returns different values for different PG versions, so we just validate that it's present and it's not `0`.